docs/BR.md

```diff
@@ -1018,31 +1018,35 @@ Certificates MUST meet the following requirements for algorithm type and key siz
 (1) Root CA Certificates
 
 ||Validity period beginning on or before 31 Dec 2010|Validity period beginning after 31 Dec 2010|
-|--|--|--|
+|---|---|---|
 |Digest algorithm|MD5 (NOT RECOMMENDED), SHA-1, SHA-256, SHA-384 or SHA-512|SHA-1\*, SHA-256, SHA-384 or SHA-512|
 |Minimum RSA modulus size (bits)|2048\*\*|2048|
 |ECC curve|NIST P-256, P-384, or P-521|NIST P-256, P-384, or P-521|
-|Minimum DSA modulus and divisor size (bits)|L= 2048 N= 224 or L= 2048 N= 256|L= 2048 N= 224 or L= 2048 N= 256|
+|Minimum DSA modulus and divisor size (bits)\*\*\*|L= 2048 N= 224 or L= 2048 N= 256|L= 2048 N= 224 or L= 2048 N= 256|
 
 (2) Subordinate CA Certificates
 
-|Digest algorithm|SHA-1*, SHA-256, SHA-384 or SHA-512|
-|Minimum RSA modulus size (bits)|2048|
-|ECC curve|NIST P-256, P-384, or P-521|
-|Minimum DSA modulus and divisor size (bits)|L= 2048 N= 224 or L= 2048 N= 256
+||Validity period beginning on or before 31 Dec 2010 and ending on or before 31 Dec 2013|Validity period beginning after 31 Dec 2010 or ending after 31 Dec 2013|
+|---|---|---|
+|Digest algorithm|SHA-1, SHA-256, SHA-384 or SHA-512|SHA-1*, SHA-256, SHA-384 or SHA-512|
+|Minimum RSA modulus size (bits)|1024|2048|
+|ECC curve|NIST P-256, P-384, or P-521|NIST P-256, P-384, or P-521|
+|Minimum DSA modulus and divisor size (bits)\*\*\*|L= 2048, N= 224 or  L= 2048, N= 256|L= 2048 N= 224 or L= 2048 N= 256
 
 (3) Subscriber Certificates
 
+||Validity period *ending* on or before 31 Dec 2013|Validity period *ending* after 31 Dec 2013|
+|---|---|---|
 |Digest algorithm|SHA-1*, SHA-256, SHA-384 or SHA-512|
```

Comment:
This row is still wrong.  The last column is blank but should be `SHA1*,	SHA‐256,	SHA‐384	or	SHA‐
512	`